docs/conf.py

```diff
@@ -106,6 +106,8 @@
 # a list of builtin themes. To override the custom theme, set this to the
 # name of a builtin theme or the name of a custom theme in html_theme_path.
 # html_theme = None
+html_static_path = ['_static', ]
+html_css_files = ['css/api.css', ]
```

Comment:
```suggestion
html_css_files = ['css/sidebar.css',]
```
This should be `sidebar.css`

Reply:
Thanks! I must have missed that